python_modules/libraries/dagster-dbt/dagster_dbt/dbt_project.py

```diff
@@ -125,8 +125,9 @@ class DbtProject(IHaveNew):
     Args:
         project_dir (Union[str, Path]):
             The directory of the dbt project.
-        target_path (Union[str, Path]):
+        output_path (Union[str, Path]):
             The path, relative to the project directory, to output artifacts.
+            In a dbt project, it corresponds to the target path.
```

Comment:
Given that this class represents a dbt project, I think it's more important to match dbt terminology than to be consistent across different technologies.

Reply:
Ya, that makes sense. I had mixed feelings about that one.